lib/jsdom/living/elements.js

```diff
@@ -0,0 +1,39 @@
+"use strict";
```

Comment:
I’d like to establish some kind of rule for what goes into this file and what goes into attributes.js. Like “things that have to do with attributes and elements but don’t require access to … module goes here”. Add this as a comment to both this file and attributes.js.

Additionally, make sure this rule is enforced.

For instance, if the rule is that anything that requires access to the generated Attr class needs to go into attributes.js, then you should say that and move everything that doesn’t need it into this file – though it looks like it would be better to move `setAttributeValue` _out_ of attributes.js in that case.

Looking further, we could probably just change `setAttributeValue` to instead use the new `document._createAttribute` to get rid of the Attr import. That might even eliminate the requirement of a separate file.